src/services/state-persistor/utils/GeometryTypes.ts

```diff
@@ -0,0 +1,56 @@
+/**
+ * @module ngeo.GeometryType
```

Comment:
ngeo? maybe remove all comments referring to the old v3 code

Reply:
removed file